lib/tasks/github_usage_tracker.rake

```diff
@@ -0,0 +1,6 @@
+namespace :github_usage_tracker do
+  desc "Record a current datapoint of GitHub API usage"
+  task :record_datapoint => :environment do
+    GithubUsageTracker.record_datapoint
+  end
```

Comment:
We do this for a couple workers already.  See https://github.com/ManageIQ/miq_bot/blob/master/app/workers/batch_job_monitor.rb#L8